geonode/maps/tests.py

```diff
@@ -168,6 +168,51 @@ def test_map_json(self):
         self.assertEquals(map_obj.abstract, "Abstract2")
         self.assertEquals(map_obj.layer_set.all().count(), 1)
 
+    @on_ogc_backend(qgis_server.BACKEND_PACKAGE)
+    def test_map_json_on_qgis_server(self):
```

Comment:
Can you merge this unittests with the above one? It seems to be doing generic things. It should be working with geoserver/qgis_server. Then we can remove `@on_ogc_backend` decorator.

Reply:
But if this truly specific to QGIS Server, then we need to move it to geonode.qgis_server.tests module